src/packages/shared-types/action-types/index.ts

```diff
@@ -4,6 +4,7 @@ export * from "./respond-to-rai";
 export * from "./withdraw-rai";
 export * from "./withdraw-package";
 export * from "./new-submission";
-export * from "./legacy-submission";
+export * from "./legacy-event";
+export * from "./legacy-package-view";
```

Comment:
this may be more trouble than its worth with our stack, but it sounds like what we're hoping to accomplish here is something `GraphQL` could help with rather than a library full of schemas